### ISSUE TITLE
new libSDL3 has still broken alsa audio (need git commit hash)

### ISSUE THREAD

**Issue by sylware** [ValveSoftware/Dota-2#2688]
The alsa fixes are not yet into upstream libSDL3.

I need the upstream libSDL3 commit hash of the currently deployed libSDL3 in order to make a build with the alsa fixes.

thx!

(meanwhile, I will try anyway to narrow down a working git commit but if you could give me the right one, would spare me some auditing work).

**Comment by sylware** [comment]
Oh, I forgot, the fixes are there:
https://github.com/libsdl-org/SDL/issues/8577

**Comment by kisak-valve** [comment]
Hello @sylware, https://github.com/ValveSoftware/csgo-osx-linux/issues/3647#issuecomment-1932384080 is equally valid here. This should be handled upstream first as maintaining the changes you want downstream doesn't make sense.

**Comment by sylware** [comment]
The fixes are still pending for testing and merging. It has been the case for months. Dunno what icculus is doing.

In the meantime:

The libSDL3 which was deployed with the birds patch is not compatible with upstream libSDL3 of today (symbols are being modified, removed).

I would need the upstream git commit hash your used for that libSDL3. I'll patch it with the fixes.

**Comment by sylware** [comment]
I found a working commit:
0d0aaa2d75265e3d9873e3a160c7ce32b311c67d

(warning: after this commit, significant symbol removals and renaming were done)

Working on publishing the fixed libSDL3 I use somewhere for download.

I did ping icculus for any news about the fixes going in. It seems very hard to get in touch with libSDL3 devs.

Because with the current code, dota2 will take exclusive access to the hardware.
